### PR TITLE
MEN-928 Clean up default device inventory.

### PR DESCRIPTION
**Pull request by pasinskim**
Remove non-static attributes and add more descriptive names for
remaining ones.

### CODE REVIEW

File: support/mender-inventory-hostinfo [hunk 1]
```diff
@@ -11,9 +11,6 @@ export LC_ALL
 grep 'model name' /proc/cpuinfo | uniq | awk -F': ' '
      // { printf("cpu_model=%s\n", $2);}
 '
-echo "cpu_count=$(ls /sys/bus/cpu/devices/ | wc -l)"
```
Comment (kacf) [inline support/mender-inventory-hostinfo]: CPU count is quite static, or?

File: support/mender-inventory-network [hunk 2]
```diff
@@ -38,7 +38,7 @@ for devpath in $SCN/*; do
         continue
     fi
     echo "mac_$dev=$(cat $devpath/address)"
-    echo "ifaces=$dev"
+    echo "network_interfaces=$dev"
```
Comment (bboozzoo) [inline support/mender-inventory-network]: you need to update example output  https://github.com/mendersoftware/mender/pull/170/commits/9c8359c4f7f49511a70128c16ae601c697a281eb#diff-971ec20e2833e1ee3573c1e1baeee0a6R18

File: support/mender-inventory-hostinfo (hunk 1 above)
Comment (pasinskim) [inline support/mender-inventory-hostinfo]: This was one of the things mentioned in acceptance criteria: https://tracker.mender.io/browse/MEN-928

File: support/mender-inventory-network (hunk 2 above)
Comment (pasinskim) [inline support/mender-inventory-network]: Thanks for spotting. Fixed!

File: support/mender-inventory-hostinfo (hunk 1 above)
Comment (pasinskim) [inline support/mender-inventory-hostinfo]: Thanks. PR here: https://github.com/mendersoftware/integration/pull/118

Comment (kacf) [inline support/mender-inventory-hostinfo]: Ok, I have added a comment there.

**Comment by pasinskim** [comment]
@kacf thanks! PR here: https://github.com/mendersoftware/integration/pull/118

File: support/mender-inventory-hostinfo (hunk 1 above)
Comment (pasinskim) [inline support/mender-inventory-hostinfo]: Sure, makes sense. Let's wait with merging this for @eysteins then.

**Comment by pasinskim** [comment]
@kacf PR is updated with the latest changes discussed in the original ticket. Related changes in https://github.com/mendersoftware/integration/pull/118 are added as well.

**Pull request merged**